### PR TITLE
chore: remove keys used by PGS for eCommerce

### PR DESCRIPTION
**Pull request by giovanniberti**
### List of changes

 * remove keys used by PGS for eCommerce
 * remove CDN rules associated to PGS

### Motivation and context

These keys were used only by transactions with XPAY and VPOS gateways which are no longer used.

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### If PR is partially applied, why? (reserved to mantainers)

Depends on pagopa/pagopa-ecommerce-event-dispatcher-service#323
Depends on pagopa/pagopa-ecommerce-transactions-service#561
Depends on #2401

### CODE REVIEW

**Review by infantesimone — CHANGES_REQUESTED** [review]
Wait :-) this API is used in prod

**Comment by infantesimone** [comment]
there is a mismatch between the PR and the description :-)

**Comment by giovanniberti** [comment]
Updated 👌

File: src/domains/ecommerce-common/02_security.tf [hunk 1]
```diff
@@ -257,18 +257,6 @@ resource "azurerm_key_vault_secret" "notifications_service_api_key" {
   }
 }
 
-resource "azurerm_key_vault_secret" "payment_transactions_gateway_api_key" {
```
Comment (infantesimone) [inline src/domains/ecommerce-common/02_security.tf:260]: this api key is used https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/aa0eabb4d0107fa92cd027cde1d6fa18e212ed6f/helm/values-prod.yaml#L191.
so this PR can be applied after the `transaction-service` no longer uses this api key, right?

Comment (giovanniberti) [reply src/domains/ecommerce-common/02_security.tf:260]: Yes, I thought I had already added the dependencies but I didn't, udpated them in the description right now :)

**Review by petretiandrea — APPROVED** [review]

**Review by infantesimone — APPROVED** [review]

**Review by infantesimone — APPROVED** [review]

**Pull request merged**